.github/workflows/scripts/start-rancher.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+flip() {
+    echo $1
+    exit 1
+}
+
+kubectl get ns | grep -s cattle && flip 'rancher already installed?'
+
+set -exu
+
+TMP="${TMP:-/tmp}"
```

Comment:
It isn't clear to me why this variable is being set. I don't see it used elsewhere in the script.